jvm/src/main/java/com/microsoft/bond/protocol/StringHelper.java

```diff
@@ -0,0 +1,40 @@
+package com.microsoft.bond.protocol;
+
+import java.nio.charset.Charset;
+
+/**
+ * Contains helper methods for working with strings.
+ */
+final class StringHelper {
+
+    private static final Charset UTF8 = Charset.forName("UTF-8");
+    private static final Charset UTF16LE = Charset.forName("UTF-16LE");
+
+    // TODO: (perf) consider using InputStream/OutputStream for encoding/decoding instead of creating temporary array buffers
```

Comment:
Let's discuss this offline. Without a TFS, TODO's seem as the only reasonable way to track backlog tasks. It's also easily discoverable whereas an issue requires more detailed documentation.

---
In reply to: [116107413](https://github.com/tstein/bond/pull/4#discussion_r116107413) [](ancestors = 116107413)